llvm/test/CodeGen/AArch64/internal-linkage-bti.ll

```diff
@@ -0,0 +1,47 @@
+; RUN: llc -mtriple=aarch64-linux-gnu %s -o - | FileCheck %s --check-prefixes=CHECK,NOBTI
```

Comment:
A better name would be `bti-something` or `branch-target-enforcement-something.ll`.
internal linkage is a minor concept and it makes sense to make it a suffix instead of a prefix to improve test discoverability.